packages/react/package.json

```diff
@@ -69,9 +69,9 @@
   "scripts": {
     "prebuild": "rimraf --glob build build-tests \"*.tsbuildinfo\"",
     "build": "pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files && pnpm build:manifest",
-    "build:node": "node ../../scripts/build.mjs node",
-    "build:stable": "node ../../scripts/build.mjs stable",
-    "build:types": "tsx ../../scripts/buildTypes.mts --project tsconfig.build.json --copy build/cjs",
+    "build:node": "node ../../scripts/build.mjs node --cjsDir ./cjs",
+    "build:stable": "node ../../scripts/build.mjs stable --cjsDir ./cjs",
+    "build:types": "tsx ../../scripts/buildTypes.mts --copy build/cjs",
```

Comment:
Does it pick up the right tsconfig automatically?

Reply:
Yes. You can also pass explicitly if it not the default which is always `tsconfig.build.json`.